packages/aws-cdk/test/integ/cli/test-cdk-nested-stack-status.sh

```diff
@@ -0,0 +1,20 @@
+#!/bin/bash
+set -euo pipefail
+scriptdir=$(cd $(dirname $0) && pwd)
+source ${scriptdir}/common.bash
+# ----------------------------------------------------------
+
+setup
+
+# verify the number of resources in the stack
+found_topic=$(mktemp)
+cdk deploy ${STACK_NAME_PREFIX}-with-nested-stack 2>&1 | grep -o -e "AWS::SNS::Topic" | uniq >> ${found_topic}
+echo "Stack deployed successfully"
+
+# assert topic status messages
```

Comment:
explain (in a comment) that the topic is defined within the nested stack and that this test verifies that the status message includes messages from the nested stacks